docs/release_notes.md

```diff
@@ -1,5 +1,46 @@
 # Release Notes
 
+## Version 1.4.0
+
+### Core
+
+- Implemented a new solver to make muscles behave more compact: AdnGlue solver.
+- Implemented Soft Constraints.
+- Added parallelization to Distance constraints.
+- Added parallelization to Shape Preservation constraints.
+- Added Hard Constraints to theAdnFat solver.
+- Added control to the minimum parallelizable chunk size via environment variable `MIN_PARALLEL_SIZE`.
+- Increased the Logger verbosity in Debug level.
+
+### Maya
+
+- Added AdnGlue node.
+- Added AdnActivation node.
+- Added support to unidirectional muscle to muscle constraints.
+- Added support to Hard Constraints in AdnFat solver including paintable map and stiffness override.
+- Optimized AdnFat deformer.
+- Optimized AdnSkin deformer.
+- Optimized AdonisFX Paint Tool.
+- Added color interpolation to the fibers debugger in AdnMuscle and AdnRibbonMuscle to encode the muscle activation.
+- Deprecated AdnWeightsDisplayNode.
+- Added a shortcut in the AdonisFX menu to upgrade deprecated nodes.
+- Added the ability to choose which constraints reinitialize at start time.
+
+
+### Bug Fixes
```

Comment:
Should we add this bugfix?: "1504 Avoid viewport refresh on deformer creation"

Reply:
I thought about it but I decided to not expose it. It was listed in the Release 1.4.0 card just for us to keep track of it internally. But actaully, it is not really but fix, I mean, nothing was "broken". The removal of that was to avoid the warning in AdnSkin saying that it could not attach to the Debugger. I

Reply:
Got it, thanks for the clarification.

Reply:
Agreed!